src/index.js

```diff
@@ -1,45 +1,127 @@
-/* eslint-disable import/no-commonjs */
-const runes = require('runes');
+import runes from 'runes';
 
-const assertIsValidText = function(text) {
+const assertIsValidText = text => {
   if (typeof text !== 'string') {
     throw new TypeError(
       'Text should be provided as first argument and be a string.'
     );
   }
 };
 
-const assertIsValidChunkSize = function(chunkSize) {
-  if (typeof chunkSize !== 'number' || chunkSize <= 0) {
+const assertIsValidChunkSize = chunkSize => {
+  if (Number.isNaN(chunkSize) || Number.parseInt(chunkSize, 10) <= 0) {
     throw new TypeError(
-      'Size should be provided as 2nd argument and be a number greater than zero.'
+      'Size should be provided as 2nd argument and parseInt to a value greater than zero.'
     );
   }
 };
 
-export default function(text, chunkSize) {
-  assertIsValidText(text);
-  assertIsValidChunkSize(chunkSize);
-
-  const chunks = [];
-  let characters = runes(text);
+const assertIsValidChunkType = (
+  chunkType,
+  chunkTypeParseIntNaN,
+  chunkTypeInt
+) => {
```

Comment:
```suggestion
function assertIsValidChunkType (
  chunkType,
  chunkTypeParseIntNaN,
  chunkTypeInt
) {
```

I don't think for example that everything should be an arrow function like this